src/test/java/com/linecorp/armeria/common/tracing/HttpTracingTestBase.java

```diff
@@ -8,7 +8,7 @@
 
 public abstract class HttpTracingTestBase {
 
-    public static final SpanId testSpanId = SpanId.create(1, 2, 3L);
+    public static final SpanId testSpanId = SpanId.builder().traceId(1).spanId(2).parentId(3L).build();
```

Comment:
/cc @adriancole FYI about the public constructor in `SpanId` which is probably meant to be package-local or private